src/views/PaymentNetworkFees.vue

```diff
@@ -0,0 +1,202 @@
+    <BaseMessageBlock
+      v-if="!web3Account"
+      level="warning"
+      class="mb-3"
+      is-responsive
+    >
+      Connect your wallet first to proceed
+    </BaseMessageBlock>
+
+    <h1>Network fees</h1>
+    <div class="text-lg mb-4">Pay to support your network on Snapshot</div>
+
+    <form v-if="fxLoadStatus === 1" class="flex flex-col gap-4" @submit="pay">
+      <fieldset>
+        <legend>
+          <h2 class="text-lg mb-2">Select your package</h2>
+        </legend>
+        <ol class="flex flex-col gap-2">
+          <li v-for="(plan, planId) in PLANS" :key="planId">
+            <label
+              class="flex gap-3 border border-skin-border rounded-xl p-3 cursor-pointer"
+              :class="{
+                '!border-skin-primary': planId === data.plan
+              }"
+              @click="setData('plan', planId)"
+            >
+              <TuneRadio :value="planId" :model-value="data.plan" />
+              <div class="flex-grow">
+                <b>{{ plan.label }}</b>
+                <BasePill
+                  v-if="plan.discount"
+                  class="ml-2 py-1 !bg-skin-primary"
+                >
+                  Save {{ plan.discount }}%
+                </BasePill>
+                <small class="block text-skin-text">
+                  {{
+                    formatFiatCurrency(computePlanFiatPrice(plan, BASE_UNIT))
+                  }}/month
+                </small>
+              </div>
+              <span>
+                {{ formatFiatCurrency(computePlanFiatPrice(plan)) }}
+              </span>
+            </label>
+          </li>
+        </ol>
+      </fieldset>
+
+      <fieldset>
+        <legend>
+          <h2 class="text-lg mb-2">Select your currency</h2>
+        </legend>
+
+        <ol class="flex flex-col gap-2">
+          <li v-for="(currency, currencyId) in CURRENCIES" :key="currencyId">
+            <label
+              class="flex border border-skin-border gap-3 rounded-xl p-3 cursor-pointer"
+              :class="{
+                '!border-skin-primary': currencyId === data.currency
+              }"
+              @click="setData('currency', currencyId)"
+            >
+              <TuneRadio :value="currencyId" :model-value="data.currency" />
+              <BaseAvatar
+                :src="snapshot.utils.getUrl(currency.logo)"
+                size="24"
+              />
+              <b class="flex-grow">{{ currency.code }}</b>
+              <small>
+                {{
+                  formatCryptoCurrency(
+                    computePrice(currencyId, PLANS[data.plan].unit),
+                    currency
+                  )
+                }}
+              </small>
+            </label>
+          </li>
+        </ol>
+      </fieldset>
+
+      <fieldset class="flex flex-col gap-2">
+        <TuneCheckbox
+          :id="'contract_signed'"
+          :model-value="data.contract_signed"
+          hint="I have signed the network contract"
```

Comment:
We discussed something about this in the call. You can watch it here https://app.fireflies.ai/view/-dev-shape-up::qb4gHWnCHfSs75BO

Reply:
This checkbox was added in response to what was said in the call. Did you mean anything else ? What else is expected ?

Reply:
Listen after 14:56

Reply:
checkbox removed